buildbot.mariadb.org/scripts/rpm-install.sh

```diff
@@ -1,79 +1,80 @@
-#!/bin/bash
+#!/usr/bin/env bash
+# shellcheck disable=SC2154
+
 set -ex
+# //TEMP why vlad?
 branch=${1:-bb-10.8-release}
```

Comment:
@vladbogo I don't understand this, we should have the same behavior as for deb IMO.

Reply:
It was something transitional before bb always passed an argument to this script. So it was something to avoid `set -ex` causing a failure before it did.

Reply:
Ok, I'll remove it then, BB should pass arguments to those script and the intelligence should be as minimal as possible in those script IMO.